activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Adds support for deferrable foreign key constraints in PostgreSQL.
+
+    Related to #17094.
```

Comment:
Probably don't need this line.

It would be cool to see an example here though.

Reply:
Yeah, you're right. Added the example from the pull request description.